### DISCUSSION TITLE
非局域网远程开机

### DISCUSSION THREAD

**Discussion by yihengM** [natfrp/wiki#235] [水深火热]
请问控制计算机和被控制计算机不在同一个局域网时如何实现远程开机？

**Comment by shenyangyeshuai** [comment]
实现不了。

主机开机是一个物理操作，无论你是按按钮，还是手动将主板的 PWM pin 触发通路，都不是远程能控制的。你只能先保证主机是运行的（并且还得开启了 SSH 服务），然后才能执行其它操作。

将隧道配置为开机自启，这样，远程重启操作后，过一会还可以 SSH 连上去，因为隧道也随着主机启动而一起启动了。

**Reply by yhw2003** [reply]
你知不知道有个东西叫网络唤醒
